repo2docker/buildpacks/base.py

```diff
@@ -682,7 +682,7 @@ def get_preassemble_scripts(self):
                     "root",
                     # This apt-get install is *not* quiet, since users explicitly asked for this
```

Comment:
Should we remove this comment? Or does it not refer to the `apt-get` below?

Reply:
@betatim the apt-get install is still not quiet. Just the apt-get update is